modules/docker/Makefile.cleanup

```diff
@@ -0,0 +1,39 @@
+## Cleanup docker volumes.             WARNING!!! IT WILL DELETE ALL UNUSED VOLUMES
+docker/cleanup/volumes:
+	@echo "Cleanup volumes"
+	@docker volume prune
+	@echo "Done"
+
+## Cleanup docker networks.            WARNING!!! IT WILL DELETE ALL UNUSED NETWORKS
+docker/cleanup/networks:
+	@echo "Cleanup networks"
+	@docker network prune
+	@echo "Done"
+
+## Cleanup docker images.              WARNING!!! IT WILL DELETE ALL UNUSED IMAGES
+docker/cleanup/images:
+	@echo "Cleanup images"
+	@docker images prune
+	@echo "Done"
+
+## Cleanup docker images all.          WARNING!!! IT WILL DELETE ALL IMAGES
+docker/cleanup/images/all:
+	@echo "Cleanup ALL images"
+	@docker images -q --no-trunc | xargs -I{} docker rmi {}
+	@echo "Done"
+
+
+## Cleanup docker containers.          WARNING!!! IT WILL DELETE ALL UNUSED CONTAINERS
+docker/cleanup/containers:
+	@echo "Cleanup containers"
+	@docker container prune
+	@echo "Done"
+
+## Cleanup docker.                     WARNING!!! IT WILL DELETE ALL UNUSED RESOURCES
+docker/cleanup:
```

Comment:
the standard `make` target is `clean`, so i think we should call this `docker/clean`. also, rename the `cleanup` namespace to `clean`

Reply:
(standard target names are `all`, `default`, `build`, `clean`, `install`, `deps`)